### PR TITLE
inventoryBuilder memory leak fix

### PR DESCRIPTION
**Pull request by viennaa**
See commit message for further insights

### CODE REVIEW

File: InventoryBuilder.py [hunk 1]
```diff
@@ -221,8 +221,8 @@ def query_vrops(self, target, vrops_short_name, iteration):
         vcenter = self.create_vcenter_objects(vrops, target, token)
         nsxt_adapter = self.create_nsxt_objects(vrops, target, token)
 
-        self.vcenter_dict[vrops] = vcenter
-        self.nsxt_dict[vrops] = nsxt_adapter
+        self.vcenter_dict[vrops_short_name] = vcenter
+        self.nsxt_dict[vrops_short_name] = nsxt_adapter
```
Comment (richardtief) [inline InventoryBuilder.py:225]: I changed all `vrops` naming in that function previously to `target`. I missed to do it for those two lines. 

Here is the commit: https://github.com/sapcc/vrops-exporter/commit/89edf43ff205c1cafadefc99d90149280abfb65f#



So I would suggest to name it `target` here too.

**Review by richardtief — CHANGES_REQUESTED** [review]

File: InventoryBuilder.py (hunk 1 above)
Comment (richardtief) [reply InventoryBuilder.py:225]: And thanks for catching this bug. Appreciated.

Comment (viennaa) [reply InventoryBuilder.py]: Going with target. Next time you can add a suggestion :)

**Review by richardtief — COMMENTED** [review]
Thanks!

**Review by richardtief — APPROVED** [review]

**Pull request merged**